### PR TITLE
Recycle certificate confirm dialog (EXPOSUREAPP-9841)

### PR DESCRIPTION
**Pull request by mtwalli**
- Adjust DGC removal dialog

### Testing
1- Scan TC, VC, RC
2- Recycle them 
3- Scan them again
4- You should be asked on scanning that TC, VC, RC are in 🗑️ and you can restore them 
5- Check navigation to details screen 

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9841

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml [hunk 1]
```diff
@@ -19,4 +19,13 @@
     <!-- XTXT: Recycle Bin  - Restore DGC dialog message -->
     <string name="recycle_bin_restore_dgc_dialog_message">"Der QR-Code wurde bereits auf Ihrem Smartphone registriert. Sie können das Zertifikat aus dem Papierkorb wiederherstellen. "</string>
 
+    <!-- DGC  - Recycle certificate dialog-->
+    <!-- XHED: Recycle Bin - recycle certificate dialog title -->
+    <string name="recycle_bin_recycle_certificate_dialog_title">"Wollen Sie das Zertifikat entfernen?"</string>
+    <!-- XTXT: Recycle Bin - recycle certificate dialog message -->
+    <string name="recycle_bin_recycle_certificate_dialog_message">"Das Zertifikat wird in den Papierkorb verschoben. Den Papierkorb finden Sie auf der Status-Registerkarte unter den drei Punkten oben rechts.Dort können Sie es endgültig löschen oder wiederherstellen. Nach 30 Tagen wird das Zertifikat automatisch gelöscht."</string>
+    <!-- XBUT: Recycle Bin - recycle certificate dialog positive button -->
+    <string name="recycle_bin_recycle_certificate_dialog_positive_button">"Zertifikat EntFernen"</string>
```
Comment (NataliaLemmerth) [inline Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:28]:
```suggestion
    <string name="recycle_bin_recycle_certificate_dialog_positive_button">"Zertifikat Entfernen"</string>
```

**Review by NataliaLemmerth — CHANGES_REQUESTED** [review]
Please take over these correction proposals. Everything else is fine.

**Review by jurajkusnier — APPROVED** [review]
LGTM

**Review by BMItr — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**